### DISCUSSION TITLE
AudioOutputI2SNoDAC output on QT Py RP2040

### DISCUSSION THREAD

**Discussion by kcriqui** [earlephilhower/ESP8266Audio#586] [Q&A]
I want to play WAV files from a littleFS filesystem on a QT Py RP2040 board. I have a short test program that seems to work, but I don't see the signal on any of the pins. Does ESP8266Audio support I2S on the RP2040?

**Comment by knuf-kandler** [comment]
Yes, it does. 

Try basic examples WAV->I2S or MOD->I2S from PROGMEM first. 

default pinout is bclk:26, lrclk:27, din:28

i can get clean output via I2S or to PWM(pins 0,1) when using progmem only. output gets distorted when loading files from LittleFS.
